mbedtls-sys/build/config.rs

```diff
@@ -421,25 +399,28 @@ pub fn default_defines() -> HashMap<&'static str, Macro> {
 pub const FEATURE_DEFINES: &'static [(&'static str, CDefine)] = &[
     ("time",                  ("MBEDTLS_HAVE_TIME",                         Defined)),
     ("time",                  ("MBEDTLS_HAVE_TIME_DATE",                    Defined)),
-    ("havege",                ("MBEDTLS_HAVEGE_C",                          Defined)),
     ("threading",             ("MBEDTLS_THREADING_C",                       Defined)),
-    ("pkcs11",                ("MBEDTLS_PKCS11_C",                          Defined)),
-    ("zlib",                  ("MBEDTLS_ZLIB_SUPPORT",                      Defined)),
     ("debug",                 ("MBEDTLS_DEBUG_C",                           Defined)),
     ("custom_printf",         ("MBEDTLS_PLATFORM_C",                        Defined)),
     ("custom_printf",         ("MBEDTLS_PLATFORM_PRINTF_MACRO",             DefinedAs("mbedtls_printf"))),
     ("aesni",                 ("MBEDTLS_AESNI_C",                           Defined)),
     ("padlock",               ("MBEDTLS_PADLOCK_C",                         Defined)),
     ("custom_has_support",    ("MBEDTLS_CUSTOM_HAS_AESNI",                  Defined)),
     ("custom_has_support",    ("MBEDTLS_CUSTOM_HAS_PADLOCK",                Defined)),
-    ("legacy_protocols",      ("MBEDTLS_SSL_PROTO_SSL3",                    Defined)),
-    ("legacy_protocols",      ("MBEDTLS_SSL_PROTO_TLS1",                    Defined)),
-    ("legacy_protocols",      ("MBEDTLS_SSL_PROTO_TLS1_1",                  Defined)),
-    ("legacy_protocols",      ("MBEDTLS_SSL_CBC_RECORD_SPLITTING",          Defined)),
     ("aes_alt",               ("MBEDTLS_AES_ENCRYPT_ALT",                   Defined)),
     ("aes_alt",               ("MBEDTLS_AES_DECRYPT_ALT",                   Defined)),
     ("mpi_force_c_code",      ("MBEDTLS_MPI_FORCE_C_CODE",                  Defined)),
     ("trusted_cert_callback", ("MBEDTLS_X509_TRUSTED_CERTIFICATE_CALLBACK", Defined)),
+    ("tls13",                 ("MBEDTLS_PSA_CRYPTO_C",                                       Defined)),
+    ("tls13",                 ("MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG",                            Defined)),
```

Comment:
I think this should only be enabled on platforms that don't have platform entropy (e.g. SGX)

Reply:
Or perhaps the `undefined` below already takes care of that?